LICENSES/MIT.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) <year> <copyright holders>
```

Comment:
Missing info?

Reply:
No, the license files themselves are always the templates and shall not be changed. Else the reuse checks fail. The year and name is part of the SPDX headers.

So the license is files are just to provide the licensed texts